hypha/apply/funds/workflow.py

```diff
@@ -380,6 +382,134 @@ def make_permissions(edit=None, review=None, view=None):
     },
 ]
 
+SingleStageReviewDefinition = [
+    {
+        DRAFT_STATE: {
+            "transitions": {
+                INITIAL_STATE: {
+                    "display": _("Submit"),
+                    "permissions": {UserPermissions.APPLICANT},
+                    "method": "create_revision",
+                },
+            },
+            "display": _("Draft"),
+            "stage": RequestRev,
+            "permissions": applicant_edit_permissions,
+        }
+    },
+    {
+        INITIAL_STATE: {
+            "transitions": {
+                "rev_more_info": _("Request More Information"),
+                "rev_review": _("Open Review"),
+                "rev_determination": _("Ready For Determination"),
+                "rev_rejected": _("Dismiss"),
+            },
+            "display": _("Need screening"),
+            "public": _("Application Received"),
+            "stage": RequestRev,
+            "permissions": default_permissions,
+        },
+        "rev_more_info": {
+            "transitions": {
+                INITIAL_STATE: {
+                    "display": _("Submit"),
+                    "permissions": {
+                        UserPermissions.APPLICANT,
+                        UserPermissions.STAFF,
+                        UserPermissions.LEAD,
+                        UserPermissions.ADMIN,
+                    },
+                    "method": "create_revision",
+                },
+            },
+            "display": _("More information required"),
+            "stage": RequestRev,
+            "permissions": applicant_edit_permissions,
+        },
+    },
+    {
+        "rev_review": {
+            "transitions": {
+                "rev_post_review_discussion": _("Close Review"),
+                INITIAL_STATE: _("Need screening (revert)"),
+            },
+            "display": _("Internal Review"),
```

Comment:
Maybe only name this stage "Review" since there is only one step and it is both internal and external.